lowpass_filter_design/generate_filter.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+"""
+Generates filter coefficients and shows its frequency response
+
+author: Colin Brochtrup
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import signal
+
+input_fs = 44.1e3
```

Comment:
Some whitespace here would help.  It's not immediately clear which line this comment belongs to.

```
# comment on var1
var1 = ...

# comment on var2
var2 = ...
```

Reply:
Fixed in ddfbcfa5fa241632362249f7bf4b7a5c94ae2d21